Reportedly/Modules/Main/Home/HomePresenter.swift

```diff
@@ -61,14 +61,21 @@ extension HomePresenter: HomeViewOutput {
     }
     
     func didTapMenuOptionLogoutButton() {
-        // TODO: Logout the user and show login screen
-        view?.showToastNotification(message: "Logout menu option pressed")
+        interactor.cleanupUserSession()
+        view?.hideRightMenuBar()
+        router.showLoginEmail()
     }
 }
 
 // MARK: - HomeInteractorOutput
 
 extension HomePresenter: HomeInteractorOutput {
+    
+    func didReceieveTokenExpiredEvent() {
+        interactor.cleanupUserSession()
+        router.showLoginEmail()
```

Comment:
I think we should have a module to handle logout or we could perhaps handle it in app delegate, then we can reuse it anytime. What do you think?

Reply:
I don't really get your point about have a module to handle logout. Also, I think handling logout in HomeModule is okay since it will always alive while the user session is active, and there is no need to reuse it at the moment. As a result, we can come back and refactor when there is a need to reuse logout logic. What do you think?

Reply:
yup, you're right, we can come back later when it need 👍